src/libraries/System.Private.Xml/src/System/Xml/Serialization/Compilation.cs

```diff
@@ -139,7 +139,7 @@ internal void InitAssemblyMethods(XmlMapping[] xmlMappings)
         /// </devdoc>
         // SxS: This method does not take any resource name and does not expose any resources to the caller.
         // It's OK to suppress the SxS warning.
-        [RequiresUnreferencedCode("calls LoadFile")]
+        [RequiresUnreferencedCode("calls LoadFrom")]
```

Comment:
This still calls LoadFile, not LoadFrom

Reply:
It used to call both. Now it only calls LoadFrom. The call to LoadFile has moved to the new method below.